src/lib/pubkey/dilithium/dilithium_common/dilithium.cpp

```diff
@@ -532,7 +532,13 @@ size_t Dilithium_PublicKey::estimated_strength() const {
    return m_public->mode().nist_security_strength();
 }
 
+std::vector<uint8_t> Dilithium_PublicKey::raw_public_key_bits() const {
+   return m_public->raw_pk();
+}
+
 std::vector<uint8_t> Dilithium_PublicKey::public_key_bits() const {
+   // Technically, that's not really correct. The docstring for public_key_bits()
+   // states that it should return a BER-encoding of the public key.
    return m_public->raw_pk();
```

Comment:
For cases like this where we are currently having an equivalence between `raw_public_key_bits` and `public_key_bits` I'd prefer the latter invoke the former so this relation is clear.